### PR TITLE
[Website] add Intuit case-study

### PR DESCRIPTION
**Pull request by bullet03**
- add intuit to case-studies
- add intuit to quote-cards on homepage
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Review by AKosolapov — CHANGES_REQUESTED** [review]

File: website/www/site/content/en/case-studies/intuit.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+      </div>
+    </blockquote>
+</div>
+<div class="case-study-post">
+
+# Powering Streaming and Real-time ML at Intuit
+
+## Background
+
+[Intuit®](https://www.intuit.com/) is a global technology platform that provides a range of financial and marketing automation solutions, including [TurboTax](https://turbotax.intuit.com/), [QuickBooks](https://quickbooks.intuit.com/), [Mint](https://mint.intuit.com/), [Credit Karma](https://www.creditkarma.com/), and [Mailchimp](https://www.google.com/aclk?sa=l&ai=DChcSEwikrNa884r4AhUyGH0KHbAmA54YABABGgJwdg&sig=AOD64_3mWKPN_mVNehe7nkfUPnwSwgfSbA&q&adurl&ved=2ahUKEwjjm9C884r4AhXVIH0KHXq8AlAQ0Qx6BAgDEAE), on its mission to power prosperity around the world. Over 100 million people trust their tax preparation, small business accounting, and personal financial management to Intuit products.
```
[the remark this answers is not in the document]
Comment (bullet03) [reply website/www/site/content/en/case-studies/intuit.md]: changed

[the remark this answers is not in the document]
Comment (AKosolapov) [reply website/www/site/content/en/case-studies/intuit.md]: Thank you!

**Comment by bullet03** [comment]
@pabloem, @aijamalnk, could you, please, review current PR?

**Comment by aijamalnk** [comment]
Thank you, LGTM!

**Pull request merged**